CHANGELOG.md

```diff
@@ -95,7 +95,7 @@
 ## v4.0.0-rc.0 (2020-04-28)
 
 #### :boom: Breaking Change
-* All components are refactored to tag-less Ember components as a first stepping stone to a full Glimmer components rewrite. In most cases this should not affect you, but there are some edge cases where it might break, e.g. using `ember-test-selectors` with the legacy curly invocation syntax.
+* All components are refactored to tag-less Ember components as a first stepping stone to a full Glimmer components rewrite. In most cases this should not affect you, but there are some edge cases where it might break, e.g. using `ember-test-selectors` with the legacy curly invocation syntax or using `attributeBindings` provided by `@ember/component` like `@id`. In all cases setting them as HTML attributes using angle bracket component invocation is the recommended path forward..
```

Comment:
```suggestion
* All components are refactored to tag-less Ember components as a first stepping stone to a full Glimmer components rewrite. In most cases this should not affect you, but there are some edge cases where it might break, e.g. using `ember-test-selectors` with the legacy curly invocation syntax or using `attributeBindings` provided by `@ember/component` like `@id`. In all cases setting them as HTML attributes using angle bracket component invocation is the recommended path forward.
```

double period :wink:

Reply:
I should have known that writing pull requests from your smartphone is maybe not the best idea. 🤪